### PR TITLE
Add unit tests of LineSorterWay

### PR DESCRIPTION
**Pull request by kimgaeunnn**
### Added
- unit tests of `LineSorterWay` actions

### Checklist ✅

- [x] The code has been linted using the command `make lint`.
- [x] Unit tests have been written to ensure major functionality is not broken.

### CODE REVIEW

File: Projects/Features/LineSorter/Tests/LineSorterTests.swift [hunk 1]
```diff
@@ -3,14 +3,83 @@
...
-import LineSorter
 import LineSorterTesting
 import XCTest
 
-final class LineSorterTests: XCTestCase {
+@testable import LineSorter
 
-    func test_example() {
-        XCTAssertEqual("LineSorter", "LineSorter")
+final class LineSorterWayTests: XCTestCase {
```
Comment (DevYeom) [inline Projects/Features/LineSorter/Tests/LineSorterTests.swift:11]: Can you change the file name?

LineSorterTests.swift -> LineSorterWayTests.swift

File: Projects/Features/LineSorter/Testing/Generated/Mocks.swift [hunk 2]
```diff
@@ -1 +1,107 @@
+public enum Mocks {
+    public enum Import {
+        public static let input: String =
```
Comment (DevYeom) [inline Projects/Features/LineSorter/Testing/Generated/Mocks.swift:3]: Please put a newline between each line.

File: Projects/Features/LineSorter/Tests/LineSorterTests.swift (hunk 1 above)
Comment (kimgaeunnn) [reply Projects/Features/LineSorter/Tests/LineSorterTests.swift]: fixed ! 2b3a27882344045e984972373ceeb708a75153d0

File: Projects/Features/LineSorter/Testing/Generated/Mocks.swift (hunk 2 above)
Comment (kimgaeunnn) [reply Projects/Features/LineSorter/Testing/Generated/Mocks.swift]: fixed~ 2b3a27882344045e984972373ceeb708a75153d0

**Review by DevYeom — APPROVED** [review]

**Pull request merged**